packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -206,14 +223,18 @@ class MouseTracker extends ChangeNotifier {
   /// This function is only public to allow for proper testing of the
   /// MouseTracker. Do not call in other contexts.
   @visibleForTesting
-  void collectMousePositions() {
+  void collectMousePositions(Iterable<int> deviceIds) {
```

Comment:
we should name this something that more clearly describes what it does, like "sendMouseNotifications" or something.

Reply:
Yeah, what the function now does has changed to not match its name anymore. Done.  I worry a little that the class is also a `ChangeNotifier` because of the mouseIsConnected state change notification, and so someone reading this might think it'll call `notifyListeners`, but I couldn't think of a better name.  Maybe `callAnnotationCallbacks`?